### PR TITLE
fix typo analysis-kuromoji

### PR DESCRIPTION
**Pull request by twosom**
### Description
remove repeated word and fixed some typo

### CODE REVIEW

File: lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/JapaneseNumberFilter.java [hunk 1]
```diff
@@ -39,9 +39,9 @@
  * <p>Notice that this analyzer uses a token composition scheme and relies on punctuation tokens
  * being found in the token stream. Please make sure your {@link JapaneseTokenizer} has {@code
  * discardPunctuation} set to false. In case punctuation characters, such as ． (U+FF0E FULLWIDTH
- * FULL STOP), is removed from the token stream, this filter would find input tokens tokens ３ and ２千
```
Comment (twosom) [inline lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/JapaneseNumberFilter.java:42]: removed repeated words `tokens tokens`.

File: lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/JapaneseNumberFilter.java [hunk 2]
```diff
@@ -59,8 +59,8 @@
  *   <li>15,7 becomes 157 (be aware of this weakness)
  * </ul>
  *
- * <p>Tokens preceded by a token with {@link PositionIncrementAttribute} of zero are left left
```
Comment (twosom) [inline lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/JapaneseNumberFilter.java:62]: removed repeated words `left left`

File: lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/completion/CharSequenceUtils.java [hunk 3]
```diff
@@ -72,7 +72,7 @@ public static boolean isFullWidthLowercaseAlphabet(char ch) {
     return ch >= 0xff41 && ch <= 0xff5a;
   }
 
-  /** Convert all hiragana in a string into kanataka */
```
Comment (twosom) [inline lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/completion/CharSequenceUtils.java:75]: fix spelling `kanataka` to `katakana`

**Review by msokolov — APPROVED** [review]
Thanks, @twosom !

**Pull request merged**